1_preprocessing_data/README.md

```diff
@@ -55,3 +55,25 @@ import pybasic
 
 Follow the instructions on the [ImageJ website](https://imagej.net/software/fiji/downloads) to install Fiji.
 We use the 64-bit Linux install.
+
+## Step 3: Create Conda Environment
+
+```sh
+# Run this command to create the conda environment for NF1 segmentation
+conda env create -f 1.preprocessing_data.yml
+```
+
+## Step 4: Activate Conda Environment
+
+```sh
+# Run this command to activate the conda environment for NF1 segmentation
+conda activate preprocessing-nf1
+```
+
+## Step 5: Execute Preprocessing NF1 Data
+
+```bash
```

Comment:
Would it make sense to update this codeblock type to match the earlier ones?

```suggestion
```sh
```

Reply:
Might as well. Honestly I don't know the difference so I will change it for consistency.

Reply:
@jenna-tomkinson - Sounds good. Just in case I wanted to share: sh is slightly different than bash. See [this SO thread](https://stackoverflow.com/questions/5725296/difference-between-sh-and-bash) for some coverage of this topic. It depends on the system being used and how the system has been setup.